lib/objects/objectsInMemFileDB.js

```diff
@@ -998,18 +951,7 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
 
     _chownFile(id, name, options, callback) {
         try {
-            if (!this.fileOptions[id]) {
-                const location = path.join(this.objectsDir, id, '_data.json');
-                if (fs.existsSync(location)) {
-                    try {
-                        this.fileOptions[id] = JSON.parse(fs.readFileSync(location, 'binary'));
```

Comment:
dito

Reply:
solved by bringing together. json is not binary